### PR TITLE
Create haven60.json

### PR DESCRIPTION
**Pull request by CMMS-Freather**
add haven60 json

adding ah haven60 via support

## Description

adding ah haven60 via support
## QMK Pull Request 

https://github.com/qmk/qmk_firmware/pull/23062

## Checklist

- [x] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

**Review by dunk2k — CHANGES_REQUESTED** [review]
- add 0.25 padding
- add WKL and HHKB bottom row options
- correct key colouring

File: v3/ah/haven60.json [hunk 1]
```diff
@@ -0,0 +1,260 @@
+{
+  "name": "Atelier_Haven Haven60",
+
```
Comment (dunk2k) [inline v3/ah/haven60.json:3]:
```suggestion
```

File: v3/ah/haven60.json [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+    "ISO Enter",
+    "Split Left Shift",
+    "Split Right Shift",
+    [
+      "Bottom Row",
+      "Full",
+      "WKL",
+      "HHKB (7u)"
+    ]
+  ],
```
Comment (dunk2k) [inline v3/ah/haven60.json:24]:
```suggestion
  "keycodes": [
    "qmk_lighting"
  ],
  "menus": [
    "qmk_rgblight"
  ],
  "matrix": {
    "rows": 5,
    "cols": 14
  },
  "layouts": {
    "labels": [
      "Split Backspace",
      "ISO Enter",
      "Split Left Shift",
      "Split Right Shift",
      [
        "Bottom Row",
        "Full",
        "WKL",
        "HHKB (7u)"
      ]
    ],
```

File: v3/ah/haven60.json [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+    "keymap": [
+      [
+        {
+          "x": 2.5,
+          "c": "#777777"
+        },
+        "0,0",
+        {"c": "#cccccc"},
```
Comment (dunk2k) [inline v3/ah/haven60.json:32]:
```suggestion
        {
          "c": "#cccccc"
        },
```

**Review by dunk2k — CHANGES_REQUESTED** [review]
general formatting

**Review by dunk2k — APPROVED** [review]

**Review by Xelus22 — APPROVED** [review]

**Pull request merged**